CHANGELOG.md

```diff
@@ -1,4 +1,29 @@
 ## 3.36.0 (Unreleased)
+
+FEATURES:
+* **New Resource:** `google_active_directory_domain_trust` ([#7056](https://github.com/hashicorp/terraform-provider-google/pull/7056))
+* **New Resource:** `google_access_context_manager_service_perimeters` ([#7027](https://github.com/hashicorp/terraform-provider-google/pull/7027))
```

Comment:
Same as the other, this one should also have new resource `google_access_context_manager_access_levels`